### ISSUE TITLE
Narrator is announcing the information of 'More Actions' icon dropdown twice after closing it with escape key.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1371]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator
Username: hcltester200@outlook.com/hcltester800@outlook.com

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Turn On Narrator using 'Ctrl+Win+Enter'.
4. Navigate to the 'Get Profile' button using down arrow key.
5. Activate 'More action' button using "enter" key.
6. Press 'Esc' key to close it and observe the issue. 

**Actual Result:**
Narrator is announcing the information of 'More Actions' icon dropdown twice after closing it with escape key.

**Expected Result:**
Narrator should announce the information of 'More Actions' icon dropdown only once after closing it with escape key.

**Note:**
Issue repro with the 'Get' And 'v1.0' dropdown before closing them.
Also repro for the link edit field.

**User Impact:** 
Screen reader users will be impacted as they will get the repeated information regarding the specified control.

**MAS Reference:**
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

https://office.visualstudio.com/OC/_workitems/edit/5682165/

**Comment by ElinorW** [comment]
Tracking here
https://office.visualstudio.com/OC/_workitems/edit/5682165/
